wp/plugin/tests/test_libs/test_wp.ml

```diff
@@ -0,0 +1,577 @@
+(***************************************************************************)
+(*                                                                         *)
+(*  Copyright (C) 2018/2019 The Charles Stark Draper Laboratory, Inc.      *)
+(*                                                                         *)
+(*  This file is provided under the license found in the LICENSE file in   *)
+(*  the top-level directory of this project.                               *)
+(*                                                                         *)
+(*  This work is funded in part by ONR/NAWC Contract N6833518C0107.  Its   *)
+(*  content does not necessarily reflect the position or policy of the US  *)
+(*  Government and no official endorsement should be inferred.             *)
+(*                                                                         *)
+(***************************************************************************)
+
+open !Core_kernel
+open Bap.Std
+open OUnit2
+open OUnitTest
+
+module StringMap = Map.Make(String);;
```

Comment:
This is a little nit-pick, but why do we need the `;;` over here?

Reply:
Don't use `;;` in `.ml` files, as a rule.

Reply:
Good point! Seems to compile fine without `;;`, so I'll remove them.